plugins/org.jkiss.dbeaver.core/plugin.xml

```diff
@@ -1299,4 +1300,13 @@
             name="%org.jkiss.dbeaver.preferences.main.toolbar.customization.name"/>
     </extension>
 
+    <extension point="org.jkiss.dbeaver.notifications">
+        <notification id="commit" title="Transaction commit"/>
+        <notification id="rollback" title="Transaction rollback"/>
+        <notification id="rollback.idle" title="Transaction rollback after long idle period"/>
+        <notification id="reconnect.success" title="Datasource invalidation success"/>
+        <notification id="reconnect.failure" title="Datasource invalidation failure"/>
```

Comment:
You were the one who asked me to write datasource as data source :) Shouldn't we do here the same?